CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+## 0.22.0
+
+### Bug Fixes
+- Introduced secrets in Deployment Settings, fixed refresh and import [#123](https://github.com/pulumi/pulumi-pulumiservice/issues/123)
```

Comment:
Can you add a little more detail about what this means for the end user? Specifically that the output of secrets will now only be ciphertext.

Reply:
Updated